src/pages/map/kakao-map.js

```diff
@@ -36,9 +37,34 @@ const $kakao = (() => {
     });
   };
 
+  const drawCurrentLocationMarker = () => {
+    navigator.geolocation.getCurrentPosition(function (position) {
```

Comment:
```suggestion
    navigator.geolocation.getCurrentPosition((position) => {
```

이런 식으로도 가능합니다.
다른 곳은 전부 애로우 펑션으로 했으니 이곳도 그러면 어떨까요?

Reply:
네 애로우 펑션으로 변경 완료하였습니다